Tasks/KubernetesManifestV0/task.json

```diff
@@ -4,7 +4,7 @@
     "friendlyName": "Deploy to Kubernetes",
     "description": "Use Kubernetes manifest files to deploy to clusters or even bake the manifest files to be used for deployments using Helm charts",
     "helpUrl": "https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/kubernetes-manifest",
-    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=851275) or [see the Kubernetes documentation](https://kubernetes.io/docs/home/)",
+    "helpMarkDown": "[Learn more about this task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/kubernetes-manifest) or [see the Kubernetes documentation](https://kubernetes.io/docs/home/)",
```

Comment:
@soccerjoshj07 , Thanks for looking at the issue, but we refer fwlinks for document. I will check with doc to get fwlink for the same.

Reply:
Use this instead https://aka.ms/kuberentesmanifest